css/style.css

```diff
@@ -1,19 +1,167 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+/* HEADER */
+.header {
+    display: flex;
+}
+
+.header-image {
+    height: 50px;
```

Comment:
in this case it looks fine, but a safer way to size images is to create a parent div for an img element and set height & width on the parent. Then you can set the height & width of the img element to be 100% and use the [object-fit](https://www.w3schools.com/css/css3_object-fit.asp) property to ensure the image is scaled correctly within the parent container